pages/security/human-security/passwords/fr.md

```diff
@@ -0,0 +1,64 @@
+* LastPass (infonuagique)
+* 1Password (infonuagique et application)
+
+# Utiliser des mots de passe robustes
+
+Les mots de passe forts sont *générés de manière aléatoire*.  Mise-à-part les mots de passe pour votre appareil et de votre gestionnaire de mots de passe, tous vos mots de passe devraient être générés par le gestionnaire et devraient être d'une longueur minimale de 12 caractères.  Les mots de passe n'ont pas besoin d'être plus longs que 26 caractères.
+
+Les être humains ne créent pas de mots de passe robustes, mais les ordinateurs, eux, en créent d'excellents.  Laissez votre ordinateur générer vos mots de passe.
+
+Pour les mots de passe qu'il faut se souvenir, il y a plusieurs façons de générer de bons.  D'abord, il y a un guide [Créer des mots de passe robustes](https://ssd.eff.org/fr/module/créer-des-mots-de-passe-robustes).
+
+[Diceware](https://web.archive.org/web/20041012030451/www.gjldp.org/CHARENTAISES/article.php?id_article=4) ([Version originale en anglais](http://world.std.com/~reinhold/diceware.html)) est une façon efficace et ludique de créer des mots de passes faciles à se rappeler, mais aléatoires.  Il utilise des dès et une liste de mots.
+
+Une autre façon de créer des mots de passe forts est d'inventer une phrase loufoque que personne n'a jamais dit par auparavant et d'utiliser la première lettre ou deux de chaque mot pour construire son mot de passe tout en y mélangeant de la ponctuation et des chiffres.
+
+Il est important d'utiliser des mots de passe robustes pour tous vos comptes parce que l'accès à un seul compte sert souvent comme chemin d'accès à d'autres systèmes.  Ceci est particulièrement vrai pour les comptes courriel qui servent à la récupération de mots de passe oubliés.
+
+# Utiliser des mots de passe uniques
+
+L'emploi de mots de passe uniques est une belle façon de minimiser le risque qui accompagne l'utilisation des services technologiques tierces.  Si un service technologique est compromis et votre mot de passe pour ce service est unique, les malfaiteurs ne seront pas capable d'accéder à vos autres comptes et services.  L'emploi de mots de passe uniques veut dire que vous ne vous fiez pas au services que vous utiliser pour votre sécurité.  C'est facile si vous utilisez un gestionnaire de mots de passe.
+
+# Garder vos mots de passe secrets
+
+Même si quelqu'un dit qu'il travaille pour le département des TI ou support technique, ne lui donnez jamais votre mot de passe.  Presque tout système permet la réinitialisation des comptes pour les fins administratives.  Tout employé légitime peut utiliser ce moyen au lieu de vous demander votre mot de passe.  Ce moyende faire laisse une trace qui peut être suivi et vous prévient de la réinitialisation.  Il faut changer son mot de passe après une telle intervention, mais en ce faisant vous vous assurez que vous êtes la seule personne avec accès à vos informations numériques. 
```

Comment:
* I would replace "le département des TI ou support technique" by "le département des TI ou le support technique".
* I would replace "réinitialisation des comptes pour les fins administratives" by "réinitialisation des comptes à des fins administratives"
* There is a space missing in "Ce moyende"
* In "une trace qui peut être suivi", "suivi" takes an "e" at the end.
* I would replace "mais en ce faisant" by "mais de cette manière"

Reply:
all fixed